components/docs-chef-io/content/automate/ha_backup_restore_aws_efs.md

```diff
@@ -29,44 +29,45 @@ A shared file system is always required to create **OpenSearch** snapshots. To r
 
 ### Setting up the backup configuration
 
--   Create an EFS file system, please refer sample steps [here](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html)
+- Create an EFS file system, please refer sample steps [here](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html)
 
-#### Configuration in OpenSearch Node
+- We have to mount EFS to all the vm's manually. To do that please refer [this](https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-mount-helper-ec2-linux.html) 
```

Comment:
I think this point and the next point should be clubbed together. 
The folder structure/directory '/mnt/automate_backups/' should be created before mounting the efs. Otherwise, the mounting won't be successful.